core/chains/evm/logpoller/log_poller.go

```diff
@@ -1060,6 +1061,14 @@ func (lp *logPoller) fillRemainingBlocksFromRPC(
 	return blocksFoundFromRPC, nil
 }
 
+// IndexedLogsWithSigsExcluding returns the set difference(A-B) of logs with signature sigA and sigB, matching is done on the topics index
+//
+// For example, query for all unfulfilled requests: request log events which don't have a matching fulfillment log event.
+func (lp *logPoller) IndexedLogsWithSigsExcluding(ctx context.Context, address common.Address, eventSigA, eventSigB common.Hash, topicIndex int, fromBlock, toBlock int64, confs int, qopts ...pg.QOpt) ([]Log, error) {
```

Comment:
hm why the separate context vs just letting callers include their context in qopts like the other methods?

Reply:
Removed context in [`e2f04b0` (#8870)](https://github.com/smartcontractkit/chainlink/pull/8870/commits/e2f04b04bf86327a1c1598d65d2cf4d33cfd0629)